### PR TITLE
extobjc: wrap macro definitions in ifndef.

### PR DESCRIPTION
**Pull request by byohay**

### CODE REVIEW

File: ReactiveCocoa/extobjc/EXTScope.h [hunk 1]
```diff
@@ -99,6 +102,7 @@ static inline void rac_executeCleanupBlock (__strong rac_cleanupBlock_t *block)
 
 #define rac_strongify_(INDEX, VAR) \
     __strong __typeof__(VAR) VAR = metamacro_concat(VAR, _weak_);
+#endif
```
Comment (StatusReport) [inline ReactiveCocoa/extobjc/EXTScope.h]: Header guards should start at the beginning of the file and end near the EOF.

**Comment by StatusReport** [comment]
Fix and squash.

**Comment by byohay** [comment]
Squashed.

File: ReactiveCocoa/extobjc/EXTScope.h (hunk 1 above)
Comment (byohay) [inline ReactiveCocoa/extobjc/EXTScope.h]: Fixed.

**Pull request merged**